plugins/modules/xenserver_guest.py

```diff
@@ -301,7 +301,6 @@
     hostname: "{{ xenserver_hostname }}"
     username: "{{ xenserver_username }}"
     password: "{{ xenserver_password }}"
-    validate_certs: false
```

Comment:
I would suggest not to remove these lines from examples but leave it with a comment like "# when using false, ensure you are in a trusted network environment!"

Using `validate_certs: true` thou default, will not work out of the box in most environments as XenServer is using self signed certificates by default and users rarely change them for proper ones. XenCenter management utility never checks certificate validity for same reason.

Reply:
How about changing `hostname` to something concrete then, like `https://192.168.1.42`? (Or how exactly are hosts specified here usually?)

This module is the only xenserver_* module which has `validate_certs` at all in the examples, the others simply use `hostname: "{{ xenserver_hostname }}"` (if at all) without showing this.

Reply:
@bvitnik any hint on how `hostname` values actually look like would be very welcome!

Reply:
@felixfontein Hi, sorry for the late reply. Now that I look more closely, only first two examples have `validate_certs`. This must be some leftover from way before when I specified some random explicit IP address and `https://` scheme. At some point I changed that to variable like `{{ xenserver_hostname }}`. This was done so that the module is in line with `vmware_guest` module which has almost identical interface. I wanted for users accustomed to VMware modules to feel at home when they start using XenServer modules.

Since all the problems related to using `https://` scheme and certificate validation are covered by notes, let's remove `validate_certs` from the first two examples as they convey no meaning and can be confusing. In other words, ignore my first comment and please proceed as initially intended :).